knative/030-trigger.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: eventing.knative.dev/v1beta1
 kind: Trigger
 metadata:
```

Comment:
Where is the YAML for deploying the map app itself? Is this still missing? I'd like to use the same for aggregator.

Reply:
@gunnarmorling the map application is a `ksvc` (Knative Serving Service), and declared here:

https://github.com/matzew/kafka-knative-demo/blob/last_tweaks/knative/020-ksvc.yaml

Reply:
Ah, ok. So do you know then how this could be replaced with a CR:

```
kubectl run --image=docker.io/gunnarmorling/debezium-knative-demo-aggregator aggregator --env="QUARKUS_KAFKA_STREAMS_BOOTSTRAP_SERVERS=my-cluster-kafka-bootstrap.kafka:9092"
```